.nuspec/Microsoft.Maui.Resizetizer.targets

```diff
@@ -4,6 +4,7 @@
     <ItemGroup>
         <AvailableItemName Include="MauiAsset" />
         <AvailableItemName Include="MauiImage" />
+        <AvailableItemName Include="MauiAppIcon" />
```

Comment:
Would `@(MauiIcon)` be a better/shorter name? Using `App` makes me think you'd want to spell out `Application`, but then that is longer...

Reply:
Yep, let me do that.